git_machete/utils.py

```diff
@@ -328,3 +329,59 @@ class CommandResult(NamedTuple):
     stdout: str
     stderr: str
     exit_code: int
+
+
+class Qualifiers:
```

Comment:
Hmmm I'd extract `git_machete/annotation.py` at this point. It's a part of the domain, deserves more than being considered a part of _utils_ ;)